metadata_backend/api/operators.py

```diff
@@ -58,6 +60,71 @@ def create_metadata_object(self, type: str, data: Dict) -> None:
         LOG.info(f"Inserting file to database succeeded: {type}, "
                  f"{self.content_type}")
 
+    def query_metadata_database(self, type: str, que: MultiDictProxy) -> Dict:
+        """Create database query based on url query parameters.
+
+        Querys are mapped below to match database structure and then query
+        is sent to mongodb. Resulting query is returned.
+
+        Keys in map are url query parameters, values are corresponding
+        database query parameters.
+
+        :param type: Type of the object to read.
+        :param que: Dict containing query information
+        :raises: HTTPBadRequest if error happened when connection to database
+        and HTTPNotFound error if file with given accession id is not found.
+        """
+        query_map = {
```

Comment:
would it be possible to load it from e.g. a schemas module instead of having this in the function ?

Reply:
sure